Documentation/contributing/development/dev_setup.rst

```diff
@@ -9,6 +9,16 @@
 Development Setup
 =================
 
+Verifying your development setup
```

Comment:
Nit: Most other titles at this level in the document seem to use title case?

Reply:
Thanks, done.